scripts/bundleSize.sh

```diff
@@ -5,8 +5,8 @@
 # If updating:
 # - The min value should be 5K less than the smallest service bundle and max should be 5K greater than the largest bundle
 # - Bundle sizes can be viewed by running a production build
-min=494
-max=522
+min=502
+max=535
```

Comment:
Howcome this is going up so much?

Reply:
This adds 5k to all services except news & persian, which it adds 10/11. Howcome?

Reply:
Found it, https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-styles/src/fonts.js gained 10k and we dont tree shake it :(